src/components/CollectionDetails/index.tsx

```diff
@@ -41,13 +41,14 @@ const CollectionDetails: React.FC<CollectionDetailsProps> = ({
   const [requestModal, setRequestModal] = useState(false);
   const [is4k, setIs4k] = useState(false);
 
-  const { data, error, revalidate } = useSWR<Collection>(
-    `/api/v1/collection/${router.query.collectionId}`,
-    {
-      initialData: collection,
-      revalidateOnMount: true,
-    }
-  );
+  const {
+    data,
+    error,
+    mutate: revalidate,
```

Comment:
How about just updating the use of these in each place to be mutate instead of just renaming it here?